dist/vue-apollo.umd.js

```diff
@@ -1980,15 +1980,19 @@ function install(Vue, options) {
     return Object.assign(map, merge(toData, fromData));
   };
 
+  console.log('## CUSTOM VUE APOLLO');
```

Comment:
Did you intend to keep it?

Reply:
Damnit. I forgot to rebuild the dist files. Fixed.